src/jsonpatcherproxy.js

```diff
@@ -256,7 +277,7 @@ const JSONPatcherProxy = (function() {
     initial process;
     */
     this.pause();
-    this._isProxifyingTreeNow = true;
+    this._isProxifyingTreeNow = true; //TODO probably not needed, when I comment this out, all tests pass
```

Comment:
I think https://github.com/Palindrom/JSONPatcherProxy/blob/c461b492a4fa2e2869ba99cbe0329db3846e93cd/src/jsonpatcherproxy.js#L95 exaplins it, but we just lack tests for removing moved obj.

Reply:
I have linked your comment in https://github.com/Palindrom/JSONPatcherProxy/issues/40, to remember this.